src/test/java/dog/pawbook/logic/parser/DeleteDogCommandParserTest.java

```diff
@@ -22,7 +22,7 @@
 
     @Test
     public void parse_validArgs_returnsDeleteCommand() {
-        assertParseSuccess(parser, "1", new DeleteDogCommand(ID_FIRST_DOG));
+        assertParseSuccess(parser, "1", new DeleteDogCommand(ID_FIRST_DOG - 1));
```

Comment:
For typical dogs, I have inserted an owner so as to make all dogs have a valid owner. And that the ID of the owner is 1, therefore all dog IDs are offset by 1. The fix here should not be with the dog ID, but rather it should be the input text. You might feel that both are equivalent, but fixing it like the current way is convoluted and nobody can understand why the input is 1 yet the ID of the first dog does not match.